test/integration/testLargeResultSet.js

```diff
@@ -252,7 +252,7 @@ describe('Large result Set Tests', function ()
   });
 });
 
-describe('SNOW-743920: Large result set with ~35 chunks', function () {
+describe('Large result set with ~35 chunks', function () {
```

Comment:
I would keep the issue id for the reference

Reply:
ok